### PR TITLE
Pass version to shortcut creator

### PR DESCRIPTION
**Pull request by tejohnso**
@fjvallarino @ahmedalsudani Please review.  I figured we'd be bundling shortcut.exe with each version bundle as we do with other dependencies.

### CODE REVIEW

File: platform.js [hunk 1]
```diff
@@ -354,9 +354,10 @@ module.exports = {
       return function() {return Promise.resolve();};
     }
   },
-  createWindowsShortcut(lnkPath, exePath, args) {
+  createWindowsShortcut(version, lnkPath, exePath, args) {
     return new Promise((resolve, reject)=>{
-      ws.create(lnkPath, { target: exePath, args: args }, (err)=>{
+      var shortcutExePath = path.join(module.exports.getInstallerDir(version), "shortcut.exe");
+      ws.create(shortcutExePath, lnkPath, { target: exePath, args: args }, (err)=>{
```
Comment (ahmedalsudani) [inline platform.js]: @tejohnso so we modified our fork so it looks for shortcut.exe in `shortcutExePath`?

Comment (tejohnso) [inline platform.js]: @ahmedalsudani see https://github.com/Rise-Vision/windows-shortcuts/pull/3

**Comment by fjvallarino** [comment]
:+1:

**Comment by ahmedalsudani** [comment]
:+1:

**Pull request merged**